src/fsinfo.ts

```diff
@@ -35,76 +56,78 @@ export interface FileInfo {
     gid?: number;
     group?: string | number;
     mtime?: number;
-    content?: string;
     target?: string;
-    entries?: {
-        [filename: string]: FileInfo;
-    };
-    targets?: {
-        [filename: string]: FileInfo;
-    };
+    entries?: Record<string, FileInfo>;
```

Comment:
Note to self: https://www.typescriptlang.org/docs/handbook/utility-types.html#recordkeys-type